packages/gatsby-plugin-mdx/src/gatsby-node.ts

```diff
@@ -208,4 +271,17 @@ export const pluginOptionsSchema: GatsbyNode["pluginOptionsSchema"] = ({
       .description(
         `Configure the file extensions that gatsby-plugin-mdx will process`
       ),
+    defaultLayouts: Joi.object({})
+      .unknown(true)
+      .default({})
+      .description(`Set the layout components for MDX source types`),
+    gatsbyRemarkPlugins: Joi.subPlugins().description(
+      `Use Gatsby-specific remark plugins`
+    ),
+    mdxOptions: Joi.object({})
```

Comment:
You can use the `.keys()` to further specify the object

Reply:
Done